### PR TITLE
Add technical details and links to details

### PR DESCRIPTION
**Pull request by fabaff**
Add technical details, update title and add links to details

Source: https://electroniccats.com/store/bast-pro-mini-m0/

### CODE REVIEW

File: _board/bast_pro_mini_m0.md [hunk 1]
```diff
@@ -12,9 +12,29 @@ features:
   - Breadboard-Friendly
 ---
 
-Is she the goddess of cats? 
+The Bast Pro Mini M0 features a powerful SAMD21E chip, an ARM Cortex-M0 based microcontroller with low power consumption and high performance, makes it ideal for endless applications.
```
Comment (dhalbert) [inline _board/bast_pro_mini_m0.md:15]: Grammar fix:
```suggestion
The Bast Pro Mini M0 features a powerful SAMD21E chip, an ARM Cortex-M0 based microcontroller with low power consumption and high performance, making it ideal for endless applications.
```

File: _board/bast_pro_mini_m0.md [hunk 2]
```diff
@@ -12,9 +12,29 @@ features:
...
+* Microcontroller Arm Cortex-M0 ATSAMD21E18A with 48 MHz
+* Flash Memory 256 KB
+* SRAM Memory 32 KB
+* Operating Voltage	1.62 V – 3.63 V
+* Input Voltage 5 V (recommended)
+* 12 Digital I/O Pins
+* 10 PWM Digital I/O Pins
+* 6 Analog Input Pins
+* DC Current per I/O Pin 10 mA
+* 13 LED (builtin)
```
Comment (dhalbert) [inline _board/bast_pro_mini_m0.md:28]: Not sure where you got that. I only see one LED on the board

File: _board/bast_pro_mini_m0.md [hunk 3]
```diff
@@ -12,9 +12,29 @@ features:
...
-Is she the goddess of cats? 
+The Bast Pro Mini M0 features a powerful SAMD21E chip, an ARM Cortex-M0 based microcontroller with low power consumption and high performance, makes it ideal for endless applications.
 
-It’s Bast Pro Mini M0! With a powerful SAMD21E chip, an ARM Cortex-M0 based microcontroller with low power consumption and high performance, makes it ideal for endless applications.
+## Technical details
+
+* Microcontroller Arm Cortex-M0 ATSAMD21E18A with 48 MHz
+* Flash Memory 256 KB
+* SRAM Memory 32 KB
+* Operating Voltage	1.62 V – 3.63 V
```
Comment (dhalbert) [inline _board/bast_pro_mini_m0.md:22]: This is the chip voltage, not the board voltage. From 
https://electroniccats.com/store/bast-pro-mini-m0/

image

File: _board/bast_pro_mini_m0.md [hunk 4]
```diff
@@ -12,9 +12,29 @@ features:
...
+* Input Voltage 5 V (recommended)
+* 12 Digital I/O Pins
+* 10 PWM Digital I/O Pins
+* 6 Analog Input Pins
+* DC Current per I/O Pin 10 mA
```
Comment (dhalbert) [inline _board/bast_pro_mini_m0.md:27]: Did you count these up, or are these the chip specifications?

**Review by dhalbert — CHANGES_REQUESTED** [review]
Hi, some mild changes and questions.

File: _board/bast_pro_mini_m0.md (hunk 2 above)
Comment (fabaff) [reply _board/bast_pro_mini_m0.md]: I interpreted the entry in the "Tech Specs" listing wrong. It seems to be the pin not the number of LEDs.

File: _board/bast_pro_mini_m0.md (hunk 4 above)
Comment (fabaff) [reply _board/bast_pro_mini_m0.md:27]: That's also coming out of their "Tech Specs" listing.

File: _board/bast_pro_mini_m0.md (hunk 3 above)
Comment (fabaff) [reply _board/bast_pro_mini_m0.md]: Updated, thanks for checking.

**Pull request merged**